### ISSUE TITLE
Error building frameworks on case-sensitive filesystem

### ISSUE THREAD

**Issue by frasersdev** [audiokit/AudioKit#819]
I use a case sensitive file systems for source code and this has revealed a problem with the [build_frameworks.sh](https://github.com/audiokit/AudioKit/blob/master/Frameworks/build_frameworks.sh) script. 

On the filesystem (https://github.com/audiokit/AudioKit/tree/master/AudioKit/iOS) the Xcode project for iOS is called "AudioKit For iOS.xcodeproj", but the [build_frameworks.sh](https://github.com/audiokit/AudioKit/blob/master/Frameworks/build_frameworks.sh) script sets it as "AudioKit for iOS.xcodeproj" (line 33).

the "for" vs "For" matters...

**Comment by megastep** [comment]
Thanks for pointing that out. Most people use a case-insensitive HFS filesystem, so this issue probably doesn't show up at all for most people.

**Comment by frasersdev** [comment]
indeed, and easily worked around by putting onto an insensitive HFS filesystem. 

cheers,
F

**Comment by megastep** [comment]
This should be fixed in the upcoming 3.5.4 release though. :)

**Comment by icare-petibles** [comment]
Hi,
I use RPi3 witch librpip.
With :
```
echo 20000 > /sys/class/pwm/pwmchip0/pwm0/period
echo 5000 > /sys/class/pwm/pwmchip0/pwm0/duty_cycle
echo 1 >  /sys/class/pwm/pwmchip0/pwm0/enable
```
I have a frequency of 500 Hz and not 50 kHz.
Where is my mistake?
Thanks
